core/controllers/base.py

```diff
@@ -513,7 +513,7 @@ def handle_exception(self, exception, unused_debug_mode):
 class Error404Handler(BaseHandler):
     """Handles 404 errors."""
 
-    REQUIRE_PAYLOAD_CSRF_CHECK = False
+    REQUIRE_PAYLOAD_CSRF_CHECK = True
```

Comment:
Actually, this is the default, right? If so you could just replace the whole line with `pass`, I think.

Reply:
Yes True is the default, i’ll keep a pass statement.